packages/flame_forge2d/lib/world_contact_listener.dart

```diff
@@ -0,0 +1,55 @@
+import 'flame_forge2d.dart';
+
+/// Listens to the entire [World] contacts events.
+///
+/// It propagates contact events ([beginContact], [endContact], [preSolve],
+/// [postSolve]) to other [ContactListener]s when a [Body] or at least one of
+/// its fixtures `userData` is set to a [ContactListener].
+///
+/// If the [Body] `userData` is set to a [ContactListener] the contact events
+/// of this will be called to when any [Body]'s fixture contacts another
+/// [Fixture].
+///
+/// If instead you wish to be more specific and only trigger contact events
+/// when a specific [Body]'s fixture contacts with another [Fixture] you can
+/// set the fixture `userData` to a [ContactListener].
+///
+/// The described behaviour is a simple out of the box solution to propagate
+/// contact events. If you wish to implement your own logic you can subclass
+/// [ContactListener] and provide it to your [Forge2DGame].
```

Comment:
@spydon Notice how we could still make the other approach still exist. Then we would offer a catalogue of different "World ContactListeners". However, I think that would overcomplicate the package.